inscriptions/ISic003534.xml

```diff
@@ -191,6 +196,13 @@
     </facsimile>
     <text>
         <body>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#AL">
+                <ab>
+                    <lb n="1"/><persName type="attested"><name type="praenomen"><expan><abbr>M</abbr><ex>arcus</ex></expan></name><g type="interpunct">·</g><name type="nomen">Va<supplied reason="lost">lerius</supplied></name>                    ̣
```

Comment:
as above, add spacing between name and g and between g and name
`<g ref="#interpunct">·</g>` and not g type
add a gap after Valerius, there's no way of knowing how much more might have been there